### ISSUE TITLE
Invalid Verification Code

### ISSUE THREAD

**Issue by shreyan007** [JKetelaar/PHP-FUT-API#29]
Fatal error: Uncaught exception 'JKetelaar\fut\api\errors\login\MainLogin' with message 'Unable to login with unknown response With the following message: Incorrect verification code With error code: 295712' in C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php:381 Stack trace: #0 C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php(98): JKetelaar\fut\api\user\Login->postTwoFactorForm('https://signin....') #1 C:\xampp\htdocs\compo\src\JKetelaar\fut\api\API.php(123): JKetelaar\fut\api\user\Login->login() #2 C:\xampp\htdocs\compo\examples\login.php(8): JKetelaar\fut\api\API->login() #3 {main} thrown in C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php on line 381

Please help... I need help urgent
Thanks in advance.

**Comment by JKetelaar** [comment]
What two factor authentication method do you have enabled?

**Comment by shreyan007** [comment]
When I updated your code with the login details and running it, I get a security code provided by EA to my email. After which I updated the **bootstrap.php** 

`$totp = new \OTPHP\TOTP('FIFA', 'Security Code form EA');`

And then I get this error:

`Fatal error: Uncaught exception 'JKetelaar\fut\api\errors\login\MainLogin' with message 'Unable to login with unknown response With the following message: Incorrect verification code With error code: 295712' in C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php:381 Stack trace: #0 C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php(98): JKetelaar\fut\api\user\Login->postTwoFactorForm('https://signin....') #1 C:\xampp\htdocs\compo\src\JKetelaar\fut\api\API.php(123): JKetelaar\fut\api\user\Login->login() #2 C:\xampp\htdocs\compo\examples\login.php(8): JKetelaar\fut\api\API->login() #3 {main} thrown in C:\xampp\htdocs\compo\src\JKetelaar\fut\api\user\Login.php on line 381`

Just to be sure I'm on the right track:
1) I'm running this code on localhost/Xampp
2) Installed this plugin with composer

**Objectives:** 
 Collect Platform Data, Start Price, Buy It Now Price and Duration from current transfer list.

_an urgent help is highly appreciated mate.
Thanks in advance :)_

**Comment by JKetelaar** [comment]
Gotcha, I've received another issue (#27), I'll have to figure out what's wrong with this method.

If you can, I'd recommend to use TOTP in the meantime

**Comment by shreyan007** [comment]
Can you tell me how to use the TOTP method

**Comment by JKetelaar** [comment]
Just released V0.0.5, which seems to work for me using Email authentication.
Can you test this for me with your account?

**Comment by nco-webdev** [comment]
@JKetelaar example would be appreciated. I've tried with my account and got 
`With the following message: Login failed`
`With error code: 295712`

Regards

**Comment by JKetelaar** [comment]
Seems to be fixed for #27 

Example code would be like:
```
<?php
require_once('vendor/autoload.php');

define('DATA_DIR', __DIR__ . '/data/');

$api = new \JKetelaar\fut\api\API('your@email.me', 'password', 'secret', 'email_callback', 'platform');

if($api->login() === true) {
    echo('We\'re logged in!' . "\n");

    $handler = $api->getHandler();
    foreach($handler->getTradepile() as $trade) {
        // Interact with $trade here
    }
}

function email_callback() {
    return 'YOUR_EMAIL_CODE';
}
```
